base/src/main/java/uk/ac/ebi/atlas/model/card/CardModelAdapter.java

```diff
@@ -53,4 +51,12 @@ private static JsonObject getTextAndUrl(String text, Optional<String> url) {
 
         return textAndUrl;
     }
+
+    private static JsonObject getTextAndUrl(Pair<Optional<String>, Optional<String>> description) {
```

Comment:
Hmm there's a bit of repeated logic here when with `getLeft` and `getRight`. Could you not pass two `Optionals` to this method, the text and the url, rather than the `Pair`? That was it will also be consistent with the other implementation of the `getTextAndUrl` method.